vm-setup/roles/common/defaults/main.yml

```diff
@@ -63,3 +63,4 @@ networks:
       forwarders:
         - domain: "apps.{{ cluster_domain }}"
           addr: "{% if baremetal_network_cidr|ipv6 != False %}::1{% else %}127.0.0.1{% endif %}"
+      srvs: "{{dns_externalsrvs | default([])}}"
```

Comment:
Note for other reviewers, the support for this already exists here:

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/libvirt/templates/network.xml.j2#L65